### ISSUE TITLE
Compilation for Mac osx

### ISSUE THREAD

**Issue by dkstewart** [VeraLiconaResearchGroup/Minimal-Hitting-Set-Algorithms#2]
Hello, I am attempting to obtain a Mac compatible executable for an x86_64 architecture (10.15.2). Using the makefile provided is not compiling. Do you have a mac compatible exe or instructions on how to obtain one? Thank you!

**Comment by veralicona** [comment]
Hello,
We don't have a Mac compatible executable. You'll have to download the source code and compile it yourself in order to make the code run. The instructions for compiling are provided in the Minimal-Hitting-Set-Algorithms repository.

**Comment by dkstewart** [comment]
Yes, I have been trying to use those instructions for compilation, but they are specific to linux. If this has been done by your group on a Mac, I would be interested in getting those specific instructions for compilation. I've been googling the compilation issues and getting nowhere...

**Comment by agdphd** [comment]
Hmm, I *think* we had this working on OSX at some point, but it was a few years ago so I don't think I still have any artifacts from that. I'll see if I can round up a Mac to test it on. If there's any specific errors you can point to or paste in in the mean time, we can see if that jogs my memory.
